templates/dashboardTemplate.Rmd

```diff
@@ -213,23 +213,30 @@ get_release_synid <- function(release_folder_fileview_synid, release) {
 }
 
 get_latest_public_release <- function(release) {
-  major_release = unlist(strsplit(release, "[.]"))[1]
-  public_major_release = as.numeric(major_release) - 1
-  public_rel = synTableQuery(paste0(
-    "select * from syn22233011 where ",
-    sprintf("name like 'Release %s.", public_major_release),
-    "%-public' order by name DESC"
-  ))
-  public_reldf = public_rel$asDataFrame()
-  nrow_df = nrow(public_reldf)
-  if (nrow_df == 0) {
-    stop(sprintf("Could not find public release for: %s", public_major_release))
-  } else if (nrow_df == 1) {
-    public_reldf$id
-  } else {
-    # The releases are ordered by latest first, so
-    # always take the first element
-    public_reldf$id[1]
+  # HACK: until we create tables for these hardcoded synids in
+  # TEST and STAGING projects, will have to return staging 
+  # version of the public release folder here
+  if (grepl("STAGING", release)){
```

Comment:
Hmm I thought we had a test public release folder, does this mean our testing pipeline now writes to this?

Reply:
Edit: nevermind, we take this code to calculate two things
1. retraction
2. number of samples added

So the dashboard won't be fully reliable to compare since couple of things

1. We don't run the staging pipeline every consortium release
2.  The test pipeline doesn't have the concept of different consortium releases.

Reply:
We do have a test public release folder but we don't have testing for dashboard in our test project. So this doesn't run for testing, [see here](https://github.com/Sage-Bionetworks/Genie/blob/9295c5c2aec47a8a62282e26eac56736a4925edb/bin/database_to_staging.py#L289-L296)